experiments/LSM/ozark/ozark_met_drivers_FLUXNET.jl

```diff
@@ -114,7 +114,15 @@ function zenith_angle(
 ) where {FT}
     # This should be time in UTC
     dt = DateTime("2004-01-01-06", "yyyy-mm-dd-HH") + Dates.Second(t)
-    FT(instantaneous_zenith_angle(dt, longitude, latitude, insol_params)[1])
+    FT(
+        instantaneous_zenith_angle(
+            dt,
+            Insolation.OrbitalData(),
```

Comment:
`Insolation.OrbitalData()` is an expensive call, so you may want to pass this variable in (let's call in `orbital_data`) if this function is called in a loop or across points in space.

Reply:
Thank you. Based on the error message though
```
UndefVarError: OrbitalData not defined
```
I think `OrbitalData` is not exported from the Insolation module. Can we do that first?

Reply:
Let's just qualify it for now.

Reply:
It was already qualified. The issue was that the `test` env was still using Insolation@0.3.1. Now I have updated it to use 0.4.1. 
The call to `OrbitalData()` happens only in the initialization. I tried to pass it in as an argument but it would produce too many changes.

Reply:
> I tried to pass it in as an argument but it would produce too many changes.

that may just be needed for this PR